po/de.po

```diff
@@ -1188,13 +1230,30 @@ msgid ""
 "to select among several online or local services from/to where you can "
 "import or export your tasks."
 msgstr ""
+"❗ Achtung: Seit GTG 0.4 wurden die Erweiterungen zu den "
+"Synchronisierungsdienste deaktiviert bis Jemand (Sie?) repariert und pflegt, "
```

Comment:
"bis sie jemand" instead of "bis Jemand" for grammatical correctness.
The uppercase "(Sie?)" refers to the person (you) while the lowercase "sie" refers to the synchronization services.